package.json

```diff
@@ -53,7 +53,9 @@
     "typescript": "4.6.3"
   },
   "dependencies": {
-    "@accordproject/concerto-util": "3.9.1"
+    "@accordproject/concerto-core": "3.14.0",
```

Comment:
This only needs to be a dev dependency.

Note that this also creates a circular dependency between concerto-core and concerto-metamodel.